src/user/dashboard/news-feed/news-feed.js

```diff
@@ -219,7 +220,7 @@ function NewsFeed(props) {
                                 <h3>{project?.projectName}</h3>
                                 <p>By {project?.projectOwner || "CODEUINO"}</p>
                                 <div className="view-project">
-                                    <Button className="view-project-btn">View Project</Button>
+                                    <Btn className="view-project-btn">View Project</Btn>
```

Comment:
Hey @ksraj123,
We are using react-bootstrap throughout the platform. So we should use the same for the button itself and then customize it as per our design for the whole platform not only dashboard.

Reply:
The `Btn` component is based on react-bootstrap `Button` component, Please have a look at `src/common/Button.js` its just a wrapper around that which makes sure the styling attributes which matter to us (i.e. `fontFamily`, `boxShadow`, `borderRadius`, `backgroundColor` and `color`)  for maintaining uniformity in style cannot be changed through styles passed through props or external stylesheets. The other way to do it would be to manually change css across the project but we would have to rely on new developers to be diligent to ensure that new buttons added are styled similarly. I felt having a common component which could be used to get a button as per the style requirements of the project would be an elegant solution to this. Thanks. Would love to know what you think.

Reply:
Also this PR is changing all the button to rectangular shape, but that's not expected, like look  at the tags of notification it should not be rectangular. Also that tabs in dashboard

Reply:
Thanks for the feedback, this is exactly the reason why I made changes only to the dashboard initially as the Issue said `"Convert all the buttons from the round shape to the rectangle shape"`. I will revert the tags of notification and tabs to as they were. It would be great if @jaskiratsingh2000 who opened the issues could comment on this as well. Thanks.

Reply:
> But as the classes are already defined there, so you can simply modify there, because component style will overwrite that.

I am not sure what you mean by this. If you are saying that the user of the Btn component could overwrite the styles in this manner <Btn style={ something.. } /> then that's not possible as in our Btn component we are overriding styles passed to it  trough props and replacing it with our required styling. Thanks.

Reply:
> Also this PR is changing all the button to rectangular shape, but that's not expected, like look at the tags of notification it should not be rectangular. Also that tabs in dashboard

@Rupeshiya Thank you for the review and suggestions,

Restored changes made to tags and section buttons, also aligned things a little, please see if it looks good now.

> But as the classes are already defined there, so you can simply modify there, because component style will overwrite that.

Also, about the `Btn` component should I scrap that and just modify styles everywhere? Thanks.

![Screenshot from 2020-07-09 04-30-1911](https://user-images.githubusercontent.com/39027928/86979324-9fe80d00-c19e-11ea-9758-df0e3dc511b0.png)